### PR TITLE
[Doc] Add *Aerospike sink connector guide*

### PR DESCRIPTION
**Pull request by Anonymitaet**
Fix https://github.com/apache/pulsar/issues/5015

### CODE REVIEW

File: site2/docs/io-aerospike-sink.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+|------|----------|----------|---------|-------------|
+| `seedHosts` |String| true | 3000 | The comma-separated list of one or more Aerospike cluster hosts.<br><br>Each host can be specified as a valid IP address or hostname followed by an optional port number. | 
+| `keyspace` | String| true | |The Aerospike namespace. |
+| `columnName` | String | true| |The Aerospike column name. |
+|`userName`|String|false||The Aerospike username.|
+|`password`|String|false||The Aerospike password.|
+| `keySet` | String|false | | The Aerospike set name. |
+| `maxConcurrentRequests` |int| false | 100 | The maximum number of concurrent Aerospike transactions that a sink can open. |
+| `timeoutMs` | int|false | 100 | This property controls `socketTimeout` and `totalTimeout` for Aerospike transactions.  |
+| `retries` | int|false | 1 |The maximum number of retries before aborting a write transaction to Aerospike. |
```
Comment (Anonymitaet) [inline site2/docs/io-aerospike-sink.md]: @tuteng what are the default values for line 18 to 22? 
I don't find them in [AerospikeSinkConfig.java](https://github.com/apache/pulsar/blob/master/pulsar-io/aerospike/src/main/java/org/apache/pulsar/io/aerospike/AerospikeSinkConfig.java)

File: site2/docs/io-aerospike-sink.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+## Configuration
+
+The configuration of the Aerospike sink connector has the following properties.
+
+### Property
+
+| Name | Type|Required | Default | Description 
+|------|----------|----------|---------|-------------|
+| `seedHosts` |String| true | 3000 | The comma-separated list of one or more Aerospike cluster hosts.<br><br>Each host can be specified as a valid IP address or hostname followed by an optional port number. |
```
Comment (tuteng) [inline site2/docs/io-aerospike-sink.md]: The default value for this option seems incorrect.
This is a required option. Its value is similar to `ip1:port1,ip2:port2,ip3:port3`, I think there is no need to add default values here.

File: site2/docs/io-aerospike-sink.md (hunk 1 above)
Comment (tuteng) [reply site2/docs/io-aerospike-sink.md]: `userName` default value is `null`, `password` default value is `null`
`keySet` default value is `null`

File: site2/docs/io-aerospike-sink.md (hunk 2 above)
Comment (Anonymitaet) [reply site2/docs/io-aerospike-sink.md]: OK

**Comment by Anonymitaet** [comment]
run cpp tests

**Comment by Anonymitaet** [comment]
@tuteng I've incorporated all your comments, could you PTAL?

**Comment by Anonymitaet** [comment]
run cpp tests

**Comment by Anonymitaet** [comment]
run cpp tests

**Comment by tuteng** [comment]
run java8 tests

**Comment by Anonymitaet** [comment]
run java8 tests

**Comment by Anonymitaet** [comment]
run java8 tests

**Comment by Anonymitaet** [comment]
run java8 tests

**Comment by Anonymitaet** [comment]
run java8 tests

**Comment by Anonymitaet** [comment]
run java8 tests

**Comment by Anonymitaet** [comment]
run java8 tests

**Pull request merged**